galleries/examples/shapes_and_collections/line_collection.py

```diff
@@ -3,79 +3,67 @@
 Plotting multiple lines with a LineCollection
 =============================================
 
-Matplotlib can efficiently draw multiple lines at once using a
-`~.LineCollection`, as showcased below.
+Matplotlib can efficiently draw multiple lines at once using a `~.LineCollection`.
 """
 
 import matplotlib.pyplot as plt
 import numpy as np
 
 from matplotlib.collections import LineCollection
 
-x = np.arange(100)
-# Here are many sets of y to plot vs. x
-ys = x[:50, np.newaxis] + x[np.newaxis, :]
+colors = ['violet', 'blue', 'green', 'yellow', 'orange', 'red']
```

Comment:
As an almost rainbow, this appears to be missing `indigo`.

Reply:
I've replaced violett by indigo. IMHO this is a more consistent look as violett is too light. That said, I've not fine-tuned the colors for a very natural look but just choosen simple color names. If somebody wants to improve the colors, they are welcome.